cloudbuild.yaml

```diff
@@ -37,6 +37,14 @@ steps:
         ".",
       ]
 
+  - name: "us.gcr.io/$PROJECT_ID/symbolicator:latest"
+    args: ['--version']
+    id: "smoke test 1"
```

Comment:
That's nice 👍

Reply:
@BYK Just to confirm, are you sure that the recently built image will be used, and not the remote image with the tag `latest`?

Reply:
^

Reply:
Since we tag the image locally, that should be the one used. There are no clear examples but this page suggests what I do is "the right way": https://cloud.google.com/cloud-build/docs/configuring-builds/build-test-deploy-artifacts#running_unit_tests_and_integration_tests

Reply:
Cool